services/src/test/scala/cromwell/pipeline/service/AuthServiceTest.scala

```diff
@@ -48,9 +47,10 @@ class AuthServiceTest extends WordSpec with Matchers with MockFactory {
 
       "return None for another type of token" taggedAs (Service) in {
         val currentTimestamp = Instant.now.getEpochSecond
-        val accessTokenContent: AuthContent = AccessTokenContent(userId = userId.value)
+        val accessTokenContent =
```

Comment:
You put serialized version of json. But what if AccessTokenContent will be renamed or moved to another place? In this case we can face with some problems. 
Why do use type as JSON field?

Reply:
We can create methods for searching pathes instead, but renaming case classes? For what?

Reply:
I guess in case of `accessToken` we shouldn't change `Json.stringify` with string. 
Access token is generated by our system and read by our system. So if there will be any changes in `AccessToken` class or json representation -- our system will know about it for sure. No other victims.

Reply:
@Dmitrii-Shevchenko Although the renaming of an existing class is a rare case, it can be moved to another package. In that case, the classpath will change and so will serialization. Moreover, the full classpath is our implementation detail, we should not share it with another world.
@ilyaonishenko I think we should discuss it in person. If it is generated and read by our system only, why do we need serialization for it? :) If we do send it somewhere, then maybe we should write a proper formatter for it.